widgets/src/google/Pie.js

```diff
@@ -16,21 +16,80 @@
     Pie.prototype = Object.create(Common2D.prototype);
     
     Pie.prototype.publish("is3D", true, "boolean", "Enable 3D");
-    Pie.prototype.publish("pieHole", 0, "number", "Pie Hole Size");
+    Pie.prototype.publish("pieHole", 0, "number", "Pie Hole Size",null,{min:0,max:0.9,step:0.1});
```

Comment:
we are going to start utilizing the ext variables for stuff like this. this can be utilized for building out a slider or other element to interact with the param in the derm layer and these properties apply to the param and building interactions. so its not used anywhere here but when using the widgets these properties come in handy